panel/src/components/Forms/Login.vue

```diff
@@ -1,32 +1,31 @@
 <template>
 	<form class="k-login-form" @submit.prevent="login">
-		<div class="k-login-fields">
```

Comment:
I guess this would technically count as a breaking change for those who have custom styles for their login

Reply:
👍